### PR TITLE
Feat/actions

### PR DESCRIPTION
**Pull request by MohamedHamouGisaia**

### CODE REVIEW

**Review by mbarbet — APPROVED** [review]

File: projects/arlas-components/src/lib/components/results/result-item/result-item.component.ts [hunk 1]
```diff
@@ -75,6 +75,9 @@ export class ResultItemComponent extends ItemComponent implements OnInit {
    */
   @Input() public selectedItems: Set<string>;
 
+  @Input() public activeActions: Map<string, Set<string>>;
```
Comment (QuCMGisaia) [inline projects/arlas-components/src/lib/components/results/result-item/result-item.component.ts:78]: Missing description

File: projects/arlas-components/src/lib/components/results/result-list/result-list.component.scss [hunk 2]
```diff
@@ -119,10 +119,26 @@
...
+          }
+        }
+      }
+    }
+
+  }
+}
+
+
+tr:hover td.resultitem__cell.toggle ::ng-deep .list_active_actions {
```
Comment (QuCMGisaia) [inline projects/arlas-components/src/lib/components/results/result-list/result-list.component.scss:140]: This could be integrated in the block above

**Review by QuCMGisaia — APPROVED** [review]

**Review by sebbousquet — APPROVED** [review]

**Review by MAudelGisaia — APPROVED** [review]

File: projects/arlas-components/src/lib/components/results/result-list/result-list.component.scss (hunk 2 above)
Comment (MohamedHamouGisaia) [reply projects/arlas-components/src/lib/components/results/result-list/result-list.component.scss]: fixed

File: projects/arlas-components/src/lib/components/results/result-item/result-item.component.ts (hunk 1 above)
Comment (MohamedHamouGisaia) [reply projects/arlas-components/src/lib/components/results/result-item/result-item.component.ts]: fixed

**Pull request merged**